src/components/AddItemPage/AddItemPage.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from "react";
+import "./AddItemPage.css";
+import FileInput from "./FileInput";
+
+function AddItemPage() {
+  const [itemInfo, setItemInfo] = useState({
+    itemName: "",
+    itemDescription: "",
+    itemPrice: "",
+    itemTags: "",
+    itemImage: null,
+  });
+
+  const [isFormValid, setIsFormValid] = useState(false);
+  //잘 안되네요. ....... . . ... .....ㅠㅠ
+
+  useEffect(() => {
+    const { itemName, itemDescription, itemPrice, itemTags } = itemInfo;
+    setIsFormValid(
+      itemName.trim() !== "" &&
+      itemDescription.trim() !== "" &&
+      itemPrice.trim() !== "" &&
+      itemTags.trim() !== ""
+    );
+  }, [itemInfo]);
+
+  const handleChange = (name, value) => {
+    setItemInfo((prevInfo) => ({
+      ...prevInfo,
+      [name]: value,
+    }));
+    updateFormValidity(name, value);
+  };
+
+  const handleInputChange = (e) => {
+    const { name, value } = e.target;
+    handleChange(name, value);
+  };
+
+  const updateFormValidity = (name, value) => {
+    const { itemName, itemDescription, itemPrice, itemTags } = itemInfo;
+    const isValid =
+      itemName.trim() !== "" &&
+      itemDescription.trim() !== "" &&
+      itemPrice.trim() !== "" &&
+      itemTags.trim() !== "";
+
+    setIsFormValid(isValid);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log("submit 확인");
+    console.log(itemInfo);
+  };
+
+  return (
+    <div className="add-item-page">
+      <form className="add-item-form" onSubmit={handleSubmit}>
+        <div className="add-item-form-top">
+          <h1>상품 등록하기</h1>
+          <button
+            className="add-item-button"
+            type="submit"
+            disabled={!isFormValid}
+          >
+            등록
+          </button>
+        </div>
+        <label htmlFor="itemImage">상품 이미지</label>
+        <FileInput
+          name="itemImage"
+          value={itemInfo.itemImage}
+          onChange={handleChange}
+        />
```

Comment:
자세히보면 이런 부분들이 반복되는것같은데, `FileInput` 내부로 가져가거나 InputContainer같은 wrapper 컴포넌트를 하나 만들어주는건 어떨까요~?
```suggestion
        <FileInput
          name="itemImage"
          value={itemInfo.itemImage}
          onChange={handleChange}
          label: '상품 이미지'
        />
```